timemachine/fe/topology.py

```diff
@@ -521,7 +531,7 @@ def parameterize_nonbonded_pairlist(self, ff_q_params, ff_q_params_intra, ff_lj_
         assert pairlist_a.get_cutoff() == pairlist_b.get_cutoff()
 
         return params, potentials.NonbondedPairListPrecomputed(
-            inclusion_idxs, pairlist_a.get_beta(), pairlist_a.get_beta()
+            inclusion_idxs, pairlist_a.get_beta(), pairlist_a.get_cutoff()
```

Comment:
Oof...

Question: Is dual topology in timemachine deprecated? (since https://github.com/proteneer/timemachine/pull/869 ?)

Reply:
Good catch!

Reply:
It's still used in rbfe for minimization. It seems to produce the same minimized conformations though for the system I tested.

Reply:
Ahh, thanks! Impact is likely small if it only affects minimization.

(Other factors: I think default `beta = 2.0`, default `cutoff = 1.2`, so this had the the effect of increasing the cutoff relative to its default. The behavior of "lambda" in 4D decoupling does depend on the value of `cutoff`, so fixing this might make the lambda schedule used for `minimize_host_4d` more efficient / more similar to its assumptions...)